generated/niscope/unit_tests/mock_helper.py

```diff
@@ -59,6 +59,9 @@ def __init__(self):
         self._defaults['Disable']['return'] = 0
         self._defaults['ExportSignal'] = {}
         self._defaults['ExportSignal']['return'] = 0
+        self._defaults['FancyFetch'] = {}
```

Comment:
#695 keeps nagging at me.

Reply:
Yeah, this'll get fixed eventually, but since it won't affect the API, it has a lower priority.